Makefile

```diff
@@ -0,0 +1,38 @@
+YARPC=yarpc
+TEST_DIR=tests/unit
+
+.PHONY: create-env
+create-env:
+	rm -rf env
+	python3 -m pip install --user virtualenv
+	python3 -m venv env
+	source env/bin/activate
+
+.PHONY: install
+install:
+	pip install --upgrade pip
+	pip install -r $(TEST_DIR)/utils/requirements.txt
+
+.PHONY: test
+test: .cleanCoverage
+	pytest $(TEST_DIR)
+
+.cleanCoverage:
+	@echo 'cleaning coverage files ...'
+	rm -f .coverage
+	rm -fr htmlcov/
+
+.PHONY: open-report
+open-report: .cleanCoverage
+	pytest --cov=$(YARPC) --cov-report=term-missing --cov-report=html
+	open -a "Google Chrome" htmlcov/index.html
```

Comment:
Why Google Chrome? I'm not sure it's a good idea to do this, not everyone uses Chrome. Maybe it's a better idea to just print html file path instead?

Reply:
I agree.